test/dataframes.jl

```diff
@@ -123,6 +123,54 @@ end
     @test d ≅ @transform(df, n1 = :i .* :g, n2 = :i .* :g)
 end
 
+@testset "@transform with @byrow" begin
```

Comment:
Maybe worth putting the new tests in a separate byrow.jl file? This one is getting pretty long.

Reply:
done.